### PR TITLE
Add docker-swarm package

### PR DESCRIPTION
**Pull request by olavphibm**
Debug info RPM is not being built.

Spec file based on openSUSE:
https://build.opensuse.org/package/view_file/Virtualization:containers/docker-swarm/docker-swarm.spec

### CODE REVIEW

File: docker-swarm/centOS/7.2/docker-swarm.spec [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+%install
+%{__mkdir_p} %{buildroot}%{_bindir}
+%{__install} -m755 swarm %{buildroot}%{_bindir}/swarm
+
+%files
+%defattr(-,root,root,-)
+%{_bindir}/swarm
+%doc LICENSE.docs README.md
+
+%changelog
```
Comment (olavphibm) [inline docker-swarm/centOS/7.2/docker-swarm.spec]: I've only changed the usual Source file name and added the line `%define debug_package %{nil}`. I couldn't make the debug info RPM build.

**Comment by olavphibm** [comment]
@maurorodrigues 
That's the only spec I've found. I've found a Koji build http://mcpkjhub1.aus.stglabs.ibm.com/koji/taskinfo?taskID=1648046, does that use a spec file?

File: docker-swarm/centOS/7.2/docker-swarm.spec (hunk 1 above)
Comment (maurorodrigues) [inline docker-swarm/centOS/7.2/docker-swarm.spec]: could you please redefine debug_package in a rpmmacro file and keep the spec as is?

**Comment by maurorodrigues** [comment]
@olavphibm it does have, and it's pretty much the same you added here. 

good to go, except by my latest comment: `could you please redefine debug_package in a rpmmacro file and keep the spec as is?`

**Comment by maurorodrigues** [comment]
closed by #58

**Pull request closed without merging**